model/tasks/ChangePermissionsTask.php

```diff
@@ -84,16 +86,20 @@ private function doHandle(
         string $requestRoot,
         array $privileges,
         bool $isRecursive,
-        bool $withNestedResources
+        bool $withNestedResources,
+        ?array $permissionsDelta
     ): Report {
+        $permissionService = $this->getPermissionService();
+        $permissionsDelta ??= $permissionService->getPermissionsDelta($root, $privileges);
```

Comment:
Just to be sure, this will work on PHP 7.4, correct?

Reply:
Yes: https://www.php.net/manual/en/migration74.new-features.php#migration74.new-features.core.null-coalescing-assignment-operator

Reply:
I think it will: https://wiki.php.net/rfc/null_coalesce_equal_operator